templates/CRM/Admin/Page/ExtensionDetails.tpl

```diff
@@ -60,15 +60,39 @@
     </tr>
     {/if}
     <tr>
-        <td class="label">{ts}Compatible with{/ts}</td>
-        <td>
-            {if $extension.compatibility}
-                {foreach from=$extension.compatibility.ver item=ver}
-                    {$ver|escape} &nbsp;
-                {/foreach}
-            {/if}
-        </td>
-    </tr>
+      <td class="label">{ts}Compatible with{/ts} (CiviCRM)</td>
```

Comment:
I think you're trying to save on translation by reusing the same string multiple times but some languages change the order of words, so these might translate oddly. I think I would just put the full thing in ts each time, in case maybe some languages don't capitalize "PHP" or use brackets differently.